lib/src/model/model.dart

```diff
@@ -38,11 +37,56 @@ class Model extends Policy {
   void loadModelFromText(String text) {}
 
   /// Saves the section to the text and returns the section text.
-  String saveSectionToText(String sec) {}
+  String saveSectionToText(String sec) => '';
 
   /// Saves the model to the text and returns the model text.
-  String saveModelToText() {}
+  String saveModelToText() => '';
 
   /// Prints the model to the log.
   void printModel() {}
+
+  @override
+  void buildRoleLinks(RoleManager rm) {}
+
+  /// printPolicy prints the policy to log.
```

Comment:
Documentation doesn't need to explicitly have the function name. So it's probably best to change it to: `/// Prints the policy to the log.`

Reply:
Also, change all of the documentation below to be consistent.